numba/np/ufunc_db.py

```diff
@@ -998,6 +1016,9 @@ def _fill_ufunc_db(ufunc_db):
     ufunc_db[np.negative].update({
         'm->m': npdatetime.timedelta_neg_impl,
     })
+    ufunc_db[np.positive].update({
+        'm->m': npdatetime.timedelta_pos_impl,
```

Comment:
Not here, but sometime it'd be good to wire in `M->M`, will probably need to patch the loop selection in `numpy_support` for that.

Reply:
That loop is deprecated:
```
In [6]: +np.datetime64('now')
<ipython-input-6-906fd83a7cf9>:1: DeprecationWarning: Applying '+' to a non-numerical array is ill-defined. Returning a copy, but in the future this will error.
  +np.datetime64('now')
Out[6]: numpy.datetime64('2020-06-11T09:26:02')
```

Reply:
ah, I had np 1.15 in my env when I checked, fair enough, this was the right choice.